src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int AMOUNT_OF_FIGURES = 5;
+    private static final double MAX_PARAMETER = 10.0;
+    private static final double DEFAULT_RADIUS = 10.0;
+
+    public Figure getRandomFigure() {
+        int randomIndex = new Random().nextInt(AMOUNT_OF_FIGURES);
```

Comment:
let's create a local variable for `new Random()` and then reuse it everywhere in this method